### PR TITLE
Explainer update

### PR DESCRIPTION
**Pull request by Valdorff**

### CODE REVIEW

**Comment by Valdorff** [comment]
Note this is still waiting on charts, but text is ready for review

File: tokenomics-explainers/003-rework-foundation.md [hunk 1]
```diff
@@ -64,30 +65,34 @@ The NO earns solo staking APY on their bond (0.334 ETH), 14% commission on the b
...
+The NO earns solo staking APY on their bond (0.334 ETH), 5% commission on the borrowed ETH (0.05 ETH), and 9% revenue share on their vote-eligible RPL (0.09 ETH)[^2]. Meanwhile, they lose RPL value to inflation going to DAO expenses (0.036 ETH). 
 
-The end result is that the Node Operator would make **0.96x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.1 ETH; that would result in **1.07x solo staking APY**. This dynamic will be explored further in the Case Study below.
+The end result is that the Node Operator would make **1.09x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.1 ETH; that would result in **1.21x solo staking APY**. This dynamic will be explored further in the Case Study below.
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:84]: Hmm... I got 1.009x solo staking, not 1.09x?
Math:
solo = (8+0.334)/8 = **4.175% APY** (conditions laid out originally were 4.17%, but we rounded to 3 digits
RP = [(8+0.334+0.05) + (2.4+0.09) - 0.036]/10.4 = 10.838/10.4 = **4.21% APY**

4.21/4.175 = **1.009x solo staking APY**

Rounded to 2 digits = **1.01x**

Comment (orangesamus) [reply tokenomics-explainers/003-rework-foundation.md:84]: Also need to update:
> voter share reward to this NO would double to 0.18 ETH; that would result in **1.22x solo staking APY**

Math:
 RP = [(8+0.334+0.05) + (2.4+0.18) - 0.036]/10.4 = 10.928/10.4 = **5.077% APY**
 5.077/4.175 = 1.216 (or rounded to 2 digits = **1.22 solo staking APY**)

Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:84]:
```suggestion
The end result is that the Node Operator would make **1.01x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.18 ETH; that would result in **1.22x solo staking APY**. This dynamic will be explored further in the Case Study below.
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 2]
```diff
@@ -100,16 +105,17 @@ Zooming _all_ the way out, the total revenue is `reth_commision*reth_tvl_in_vali
 ### ETH-only Validator ROIs (Saturn 1)
 Zooming in to a single operator, the easiest way to look at the impact of lower bonds is by considering ETH-only validators and looking at their ROI. As we can see - we get dramatically higher ROIs for the same commission (or, equivalently, can get the same ROI with dramatically lower commission). The takeaway is simple from a capital efficiency point of view – smaller bonds are better.
 
+TODO -- update circle and legend text to 5%. make x axis go from 3-7%. ideally samus has the final version of code; if not, searching for "arrowstyle" in discord gives code to make an earlier version of this chart.
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:108]: Sorry, I think you made the final tweaks here:
https://discord.com/channels/405159462932971535/1215788197842255972/1245019226385944619

Like you said, the last I had was from the "arrowstyle" message here:
https://discord.com/channels/405159462932971535/1215788197842255972/1244787553119240292

Could I let you regenerate? I could try to go back and make changes to end up in the same style, but am working on the sankey's and pie chart right now

**Comment by orangesamus** [comment]
Planning to do a separate PR shortly for the pie chart/sankeys I regenerated. Just commenting here as a reminder to follow up and delete the "TODOs" on this PR

File: tokenomics-explainers/003-rework-foundation.md [hunk 3]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+In this case, no share of ETH revenue is used to buy RPL (which means unstaked RPL does not directly benefit from the ETH revenue). Instead, the equivalent share of ETH revenue is simply used to have a larger `voter_share` which is distributed according to vote-eligible RPL staked to megapools. This means only NOs are natively able to benefit from RPL’s value capture directly. This has some positives (direct ETH received instead of indirect value captured; avoids some complexity in implementing one of the above options) and some negatives (much smaller total addressable market). Note that it may become possible for non-NOs to benefit via abstractions (eg, Nodeset’s proposed xRPL).
+
+### What would plots look like under other options?
+As we noted initially, "Voter Share" was assumed for this explainer.
+
+Here we show the pie charts for "Buy + x" on the left and "Voter Share" on the right. Note the new "Surplus Share" slice, which is drawn as blue with gold dots to show that it's going to both staked RPL held by NOs and a new beneficiary of protocol revenue for some of this slice (unstaked RPL).
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:175]:
```suggestion
Here we show the pie charts for "Buy + x" on the left and "Voter Share" on the right. Note the new "Surplus Share" slice, which is drawn as blue with gold dots. The gold dots represent the new beneficiary of protocol revenue (unstaked RPL), and the blue represents the same beneficiary as before (NOs with staked RPL). 
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 4]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+
+TODO -- make and number figure; maybe 5% voter_share and 4% surplus_share?
+<div>
+    <img src="../assets/tokenomics-explainers/003-figure-1.png" alt="Figure 1" width="1000px"></br>
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:179]:
```suggestion
    <img src="../assets/tokenomics-explainers/003-figure-9.png" alt="Figure 1" width="1000px"></br>
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 5]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+    <i>Figure - Proposed tokenomics pies for "Buy + x" and "Voter Share" respectively</i>
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:180]:
```suggestion
    <i>Figure 9 - Proposed tokenomics pies for "Buy + x" and "Voter Share" respectively</i>
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 6]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+</div>
+<br/>
+
+TODO -- make Sankeys similar to current Figure 5, one will be used above already and just be a copy. The other will be a new variant with surplus_share in it. Swap in figures and number.
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:184]:
```suggestion
Here we compare sankey diagrams of the _Case Study Reference_ example from before with a new example that includes a "Surplus Share" flow, which could be used for Buy + Burn or Buy + LP. This share would benefit all RPL (including both staked and unstaked RPL).
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 7]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+<img src="../assets/tokenomics-explainers/003-figure-6.png" alt="Figure 6" width="1000px"></br>
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:185]:
```suggestion
<img src="../assets/tokenomics-explainers/003-figure-10.png" alt="Figure 10" width="1000px"></br>
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 8]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+_Figure  - Case Study ETH-only Growth Sankey Diagram_
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:186]:
```suggestion
_Figure 10 - Case Study Reference Example_
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 9]
```diff
@@ -146,22 +154,39 @@ There are natural checks and balances that allow "non-voting" participants to cr
...
+<img src="../assets/tokenomics-explainers/003-figure-6.png" alt="Figure 6" width="1000px"></br>
+_Figure  - Case Study ETH-only Growth Sankey Diagram_
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:188]:
```suggestion
_Figure 11 - Case Study ETH-only Growth Sankey Diagram_
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 10]
```diff
@@ -20,25 +20,25 @@ depth: Intermediate
...
+Importantly, the same participant may be eligible to benefit from both categories. As an example take the Node Operator in existing tokenomics, represented in Figure 1 on the left pie chart in blue. You can see how an equivalent participant shows up in blue in the new tokenomics on the right pie chart. Their ETH earns from the Node Operator Commission Share, which is initially 5% in the current proposal. Their staked RPL earns from the Voter Share, which is initially 9% in the current proposal.
 
+TODO -- update to 5% no_share and 9% voter_share
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:34]:
```suggestion
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 11]
```diff
@@ -64,30 +65,34 @@ The NO earns solo staking APY on their bond (0.334 ETH), 14% commission on the b
 
 The end result is that the Node Operator would make **1.14x solo staking APY right now**. At maturity, RPL would approach ~100% staked, at which point RPL rewards and Inflation to NOs cancel each other out resulting in **1.01x solo staking APY**. Back to the current state, if a Node Operator loses eligibility for RPL rewards (which often happens due to falling RPL/ETH price ratio), the loss of the 0.14ETH in RPL rewards would result in **0.82x solo staking APY**.
 
+TODO -- update to new no_share
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:68]:
```suggestion
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 12]
```diff
@@ -64,30 +65,34 @@ The NO earns solo staking APY on their bond (0.334 ETH), 14% commission on the b
...
 <img src="../assets/tokenomics-explainers/003-figure-3.png" alt="Figure 3" width="1000px"></br>
 _Figure 3 - Proposal ETH-only Sankey Diagram_
 
-The NO earns solo staking APY on their bond (0.334 ETH), and 3.5% commission on the borrowed ETH (0.035 ETH).
+The NO earns solo staking APY on their bond (0.334 ETH), and 5% commission on the borrowed ETH (0.05 ETH).
 
-The end result is that the Node Operator would make **1.11x solo staking APY right now**.
+The end result is that the Node Operator would make **1.15x solo staking APY right now**.
 
+TODO -- update to new no_share and voter_share
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:77]:
```suggestion
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 13]
```diff
@@ -64,30 +65,34 @@ The NO earns solo staking APY on their bond (0.334 ETH), 14% commission on the b
...
+The NO earns solo staking APY on their bond (0.334 ETH), 5% commission on the borrowed ETH (0.05 ETH), and 9% revenue share on their vote-eligible RPL (0.09 ETH)[^2]. Meanwhile, they lose RPL value to inflation going to DAO expenses (0.036 ETH). 
 
-The end result is that the Node Operator would make **0.96x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.1 ETH; that would result in **1.07x solo staking APY**. This dynamic will be explored further in the Case Study below.
+The end result is that the Node Operator would make **1.09x solo staking APY right now**. Note that voter share depends on your exposure to RPL relative to other NOs. Eg, if half of the validators are ETH-only and half look like this NO, then the voter share reward to this NO would double to 0.1 ETH; that would result in **1.21x solo staking APY**. This dynamic will be explored further in the Case Study below.
 
 #### Case Study: ETH-only is Preferred by Node Operators
 
+TODO -- update to new no_share and voter_share
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:88]:
```suggestion
```

File: tokenomics-explainers/003-rework-foundation.md [hunk 14]
```diff
@@ -64,30 +65,34 @@ The NO earns solo staking APY on their bond (0.334 ETH), 14% commission on the b
...
 **Proposal Example, Reference**  
 <img src="../assets/tokenomics-explainers/003-figure-5.png" alt="Figure 5" width="1000px"></br>
 _Figure 5 - Case Study Reference Sankey Diagram_
 
 In this case study, we have 3 NOs. (A) has 8 ETH and no RPL staked. (B) has 8 ETH and 6 ETH worth of RPL staked. (C) has 8 ETH and 15 ETH worth of RPL staked (note that vote-eligible RPL is capped at 150% of bonded ETH. This leads to NO C earning twice as much from "Voter Share" as NO B).
 
+TODO -- update to new no_share and voter_share
```
Comment (orangesamus) [inline tokenomics-explainers/003-rework-foundation.md:95]:
```suggestion
```

**Comment by orangesamus** [comment]
I updated the charts separately, we should be able to just merge both this branch and my other branch to the main spring2024-tokenomics and everything should look fine from there I think
https://github.com/rocket-pool/RPIPs/pull/256

File: tokenomics-explainers/003-rework-foundation.md (hunk 1 above)
Comment (Valdorff) [reply tokenomics-explainers/003-rework-foundation.md:84]: I agree with these numbers - good catch

File: tokenomics-explainers/003-rework-foundation.md (hunk 2 above)
Comment (Valdorff) [reply tokenomics-explainers/003-rework-foundation.md:108]: I don't have the code any more then. Sadge. Anyhow -- regenerating

**Pull request merged**